build_web_compilers/lib/src/dev_compiler_bootstrap.dart

```diff
@@ -265,19 +265,29 @@ String _entryPointJs(String bootstrapModuleName) => '''
   var mainUri = _currentDirectory + "$bootstrapModuleName";
 
   if (typeof document != 'undefined') {
-    // Load stack trace mapper
-    var el = document.createElement("script");
-    el.defer = true;
-    el.async = false;
-    el.src = mapperUri;
-    document.head.appendChild(el);
+    var el;
+
+    // Load stack trace mapper if not already loaded
+    var mapperId = 'dart-stack-trace-mapper';
+    if (!document.querySelector('#' + mapperId)) {
+      el = document.createElement("script");
```

Comment:
might as well make `el` be a local variable to each of these blocks now instead of a shared one we overwrite